crates/plugin_css/src/dep_analyzer.rs

```diff
@@ -70,7 +76,19 @@ impl Visit for DepAnalyzer {
 pub fn is_source_ignored(source: &str) -> bool {
   source.starts_with("http://")
     || source.starts_with("https://")
-    || source.starts_with('/')
+    || source.starts_with("/")
     || source.starts_with("data:")
     || source.starts_with('#')
 }
+
+pub fn is_start_with_alias(alias: &HashMap<String, String>, source: &str) -> bool {
```

Comment:
alias may be a regex, refer to plugin resolve, i think a common alias matching util may need to be extracted to toolkit